torch/csrc/utils/tensor_new.cpp

```diff
@@ -42,6 +46,10 @@ using at::Type;
 using c10::optional;
 
 namespace torch { namespace utils {
+
+at::Tensor tensor_from_ndarray_batch(std::vector<PyArrayObject*> & array_ptr_storage, std::vector<int64_t> & sizes, ScalarType scalarType, bool pin_memory);
```

Comment:
Instead of declaring these here put them in a header.

Reply:
Ah, I actually forgot to move those into a header.